kratos/utilities/quadrature_points_utility.h

```diff
@@ -129,6 +130,32 @@ namespace Kratos
                     LocalTangentV);
         }
 
+        static GeometryPointerType CreateQuadraturePointSurfaceInVolume(
+            GeometryShapeFunctionContainer<GeometryData::IntegrationMethod>& rShapeFunctionContainer,
+            PointsArrayType rPoints,
+            Matrix LocalTangentMatrix,
+            GeometryType* pGeometryParent)
+        {
+            return Kratos::make_shared<
+                QuadraturePointSurfaceInVolumeGeometry<TPointType>>(
+                    rPoints,
+                    rShapeFunctionContainer,
+                    LocalTangentMatrix,
+                    pGeometryParent);
+        }
+
+        static GeometryPointerType CreateQuadraturePointSurfaceInVolume(
+            GeometryShapeFunctionContainer<GeometryData::IntegrationMethod>& rShapeFunctionContainer,
+            PointsArrayType rPoints,
+            Matrix LocalTangentMatrix)
```

Comment:
I changed this form BoundedMatrix to Matrix, such that we can catch an dimension error in `quadrature_point_surface_in_nurbs_volume_geometry`. Btw all the points and matrices, we could pass by reference, as they will be copied when the respective constructor is called afterwards anyway. Right now, we are copying twice, right?

Reply:
That is a good question. I think it should be passed by source, here. Also, the name `rPoints`should be either source or just `Points`. But I believe we should test once doing all those functions with passing the points by ref.

Reply:
Let's do this in a separate PR though.